src/ui/Backdrop/Backdrop.ts

```diff
@@ -0,0 +1,74 @@
+import {Component} from '../Base/Component';
+import {ComponentOptions} from '../Base/ComponentOptions';
+import {IComponentBindings} from '../Base/ComponentBindings';
+import {Initialization, IInitializationParameters} from '../Base/Initialization';
+import {IResultsComponentBindings} from '../Base/ResultsComponentBindings';
+import {IQueryResult} from '../../rest/QueryResult';
+
+export interface IBackdropOptions {
+  imageField: string;
+  overlayColor?: string;
+  overlayGradient?: boolean;
+}
+
+/**
+ * This component is used to render an image URL (contained in a result's field)
+ * as a background image.
+ */
+export class Backdrop extends Component {
+  static ID = 'Backdrop';
+
+  /**
+   * @componentOptions
+   */
+  static options: IBackdropOptions = {
+    /**
+     * Specifies the field from which the background image will be pulled.
+     */
+    imageField: ComponentOptions.buildStringOption({ required: true }),
```

Comment:
We should also support an imageUrl option (which accept a 'static' value if you wish, and not forcing the url to be contained in a field value.)

This would be useful for underscore template, where JS logic can be used to construct the value.

As such, imageField should not be required, and there should be some logic in the component to add priority if both options are set (probably imageUrl > imageField)